UseCase45/bin/setup-worker.php

```diff
@@ -17,7 +17,7 @@
     ->setWorkerMethod('start')
     ->setName('Retry Requester')
     ->setCronExpression('* * * * *')
-    ->setCanWorkInParallel(false)
+    ->setCanWorkInParallel(true)
     ->setDefaultImportance(10)
     ->setScheduleLimit(50)
```

Comment:
Should the `schedule_limit` go to `1` since you flipped the `can_work_in_parallel`?
```suggestion
    ->setScheduleLimit(1)
```

Reply:
good catch, flipping it reflects the use case, ty!